### ISSUE TITLE
Doctest ELLIPSIS explanation hard to follow when they're missing

### ISSUE THREAD

**Issue by e1477117-cfdb-4d20-89a6-df626bd6ae67** [python/cpython#87842]
BPO | [43676](https://bugs.python.org/issue43676)
--- | :---

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2021-03-31.05:31:48.136>
labels = ['type-feature', '3.9', 'docs']
title = "Doctest ELLIPSIS explanation hard to follow when they're missing"
updated_at = <Date 2021-03-31.05:31:48.136>
user = 'https://bugs.python.org/TimHatch'
```

bugs.python.org fields:
```python
activity = <Date 2021-03-31.05:31:48.136>
actor = 'Tim.Hatch'
assignee = 'docs@python'
closed = False
closed_date = None
closer = None
components = ['Documentation']
creation = <Date 2021-03-31.05:31:48.136>
creator = 'Tim.Hatch'
dependencies = []
files = []
hgrepos = []
issue_num = 43676
keywords = []
message_count = 1.0
messages = ['389874']
nosy_count = 2.0
nosy_names = ['docs@python', 'Tim.Hatch']
pr_nums = []
priority = 'normal'
resolution = None
stage = None
status = 'open'
superseder = None
type = 'enhancement'
url = 'https://bugs.python.org/issue43676'
versions = ['Python 3.9']
```

</p></details>

**Comment by StanFromIreland** [comment]
Directives are now visible in the rendered docs. (I have verified 3.13+)

<img width="1437" height="152" alt="Image" src="https://github.com/user-attachments/assets/6c8f057b-c309-4429-8737-05132a019e5d" />
